.vscode/tasks.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Build with build.py",
+            "type": "shell",
+            "command": "python build.py",
+			"windows": {
+				"command": "py build.py"
+			},
+            "problemMatcher": [],
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        },
+        {
+            "label": "Build and deploy with build.py",
+            "type": "shell",
+            "command": "python build.py -f",
+			"windows": {
+				"command": "py build.py -f"
+			},
+            "problemMatcher": [],
+            "group": {
+                "kind": "build",
+                "isDefault": false
+            }
+        },
+        {
+            "label": "Deploy",
+            "type": "shell",
+            "command": "python -c \"from flash import flash; flash()\"",
+			"windows":{
+				"command": "py -c \"from flash import flash; flash()\""
+			},
+            "problemMatcher": []
+        }
```

Comment:
Trying to execute deploy I got this:

```
Executing task: python -c "from flash import flash; flash()" <

Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'flash'
The terminal process "/bin/zsh '-c', 'python -c "from flash import flash; flash()"'" terminated with exit code: 1.

Terminal will be reused by tasks, press any key to close it.
```

Reply:
Oops! Fixed with 5c8a1f30669d6a71e08bfa4bac94d6d8a5157cc2.